content/php/concepts/math-functions/terms/pow/pow.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'pow()'
+Description: 'Returns the number raised to the power of the exponent.'
+Subjects:
+  - 'Web Development'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+  - 'Numbers'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`pow()`** function returns the number raised to the power of the exponent.
+
+## Syntax
+
+```pseudo
+pow(number $base, number $exp) : number
```

Comment:
@AryaMehrian26 What is the `: number` referring to?

Reply:
Hello. Yes, we can omit it.